downstream/assemblies/platform/assembly-ug-controller-job-templates.adoc

```diff
@@ -0,0 +1,55 @@
+[id="controller-job-templates"]
+
+= Job templates
+
+A job template is a definition and set of parameters for running an Ansible job. 
+Job templates are useful to execute the same job many times. 
+They also encourage the reuse of Ansible playbook content and collaboration between teams.
+
+The *Templates* list view shows job templates that are currently available. 
+The default view is collapsed (Compact), showing the template name, template type, and the timestamp of the last job that ran using that template. 
+You can click the arrow next to each entry to expand and view more information. 
+This list is sorted alphabetically by name, but you can sort by other criteria, or search by various fields and attributes of a template.
+
+image::ug-job-templates-home.png[Job templates home]
+
+From this screen, you can launch, edit, and copy a job template. 
```

Comment:
I've added the rocket icon rightrocket.png, for 'Launch' but it's not available yet.